src/vaadin-custom-field-mixin.html

```diff
@@ -138,5 +154,12 @@
         this.validate();
       }
     }
+
+    /**
+     * Fired on Tab keydown triggered from the internal inputs,
+     * meaning focus will not leave the inputs.
+     *
+     * @event internal-tabbing
```

Comment:
The event name doesn't sound right, compare "clicking", "focusing"

Reply:
Renamed to `internal-tab`